docs/whatsnew.rst

```diff
@@ -95,80 +95,47 @@ Technical novelties in eXo Platform 5.1
 
 **Upgrades**
 
-In the version 5.0, many tools/applications used by eXo Platform were 
-upgraded to newer  versions in order to benefit from their latest 
-updates and improvements: 
+In 5.1 version, eXo Platform upgraded to Jboss EAP 7.1 in order to:
```

Comment:
I would rephrase to "eXo Platform 5.1 now supports JBoss EAP 7.1. This new version comes with some improvements and fixes some issues which impacted eXo Platform:
* Remember me" token not recognized in Jboss EAP 7.0 (PLF-7787)
* Instance id of a cluster node is not added on the JSESSIONID cookie (PLF-7854)
"
And set links on :
* "some improvements" -> https://www.redhat.com/en/blog/red-hat-releases-jboss-eap-71
* "PLF-7787" -> https://jira.exoplatform.org/browse/PLF-7787
* "PLF-7854" -> https://jira.exoplatform.org/browse/PLF-7854

WDYT ?

Reply:
OK for the phrase but not really with putting jira's links because sometime there are internal links that may make the full sense incomplete when it is an answer to a previous comment

Reply:
The goal is to give more information about the issues fixed by this version. Putting only the short summary is not enough for the customer to know if it is the issue he face IMO.

Reply:
OK